aspnetcore/mvc/views/view-compilation.md

```diff
@@ -47,37 +47,23 @@ Runtime compilation can be enabled such that it's only available for local devel
 
 To enable runtime compilation based on the environment and configuration mode:
 
-1. Conditionally reference the [Microsoft.AspNetCore.Mvc.Razor.RuntimeCompilation](https://www.nuget.org/packages/Microsoft.AspNetCore.Mvc.Razor.RuntimeCompilation/) package based on the active `Configuration` value:
```

Comment:
Same recommendation as above. These are sequential items, so a numbered list is what the style guide prescribes.